easybuild/easyconfigs/w/Wannier90/Wannier90_1x_ignore_makesys.patch

```diff
@@ -0,0 +1,11 @@
+--- src/Makefile.orig	2016-04-19 15:26:27.373047000 +0800
```

Comment:
@migueldiascosta please include a line of comment explaining what this patch does, why it is needed, and mention yourself as an author (any lines above the `---` line are comments); please do so in both patches

```
avoid including make.sys, which contains hardcoding settings we don't need/want
author: Miguel Dias Costa (National University of Singapore)
```